appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
+$repoProvider=$env:APPVEYOR_REPO_PROVIDER
+$projectSlug=$env:APPVEYOR_PROJECT_SLUG
+$repoName=$env:APPVEYOR_REPO_NAME
+$repoCommit=$env:APPVEYOR_REPO_COMMIT
+#$repoProvider='gitHub'
+#$projectSlug='sarif-sdk'
+##$projectSlug='sarif-sdk-931m5'
+#$repoName='Rolstenhouse/sarif-sdk'
+#$repoCommit='f82ebd892917bd6157efc04b832a57964ec1f2c6'
+
+$githubBaseUri = ""
+
+if($repoProvider -eq "gitHub")
+{
+  $githubBaseUri = "https://raw.githubusercontent.com/"
+}
+
+function Rebase-Uri($originalURI)
+{
+  $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString($originalURI.IndexOf("c:/")))
+  return $githubBaseUri + $repoName+"/"+$repoCommit+"/"+$caseSensitiveUri.SubString($caseSensitiveUri.IndexOf($projectSlug)+$projectSlug.Length+1)
+}
+
+function Get-CaseSensitivePath($pathName)
+{
+  $pathExists = Test-Path($pathName)
+  if (-Not $pathExists)
+  {
+      return $pathName
+  }
+  $directoryInfo = New-Object IO.DirectoryInfo($pathName)
+
+  if ($directoryInfo.Parent -ne $null)
+  {
+      $parentPath = Get-CaseSensitivePath($directoryInfo.Parent.FullName)
+      $childPath = $directoryInfo.Parent.GetFileSystemInfos($directoryInfo.Name)[0].Name
+      return(Join-Path $parentPath $childpath -resolv)
+  }
+  else
+  {
+      return $directoryInfo.Name
+  }
+}
+
+$sarifLog = (Get-Content $location) -join "`n" | ConvertFrom-Json
+
+for ($i = 0; $i -lt $sarifLog.runs.Count; $i++)
+{
+  #files
+  for($j = 0; $j -lt $sarifLog.runs[$i].files.Count; $j++)
+  {
+    $sarifLog.runs[$i].files[$j].uri = Rebase-Uri($sarifLog.runs[$i].files[$j].uri)
+    #UNTESTED
+    $props = JSON.runs[$i].files[$j].PSObject.Properties
+    if($props.Name)
+    {
+      $URIReplacement = $props.Name
+      $content = $props.Value
+      $sarifLog.runs.files.PSObject.Properties.Remove($props.Name)
+      $sarifLog.runs.files | add-member -Name Rebase-Uri($URIReplacement) -Value $content -MemberType NoteProperty
+    }
+  }
+
+  #results
+  for($j = 0; $j -lt $sarifLog.runs[$i].results.Count; $j++)
```

Comment:
This would need to be in a loop over the `runs`, indexed by `$i`. As the code stands, you do this only for one run, using whatever value `$i` happens to have at the moment.